webpack/ForemanInventoryUpload/Components/AccountList/Components/ListItem/ListItem.js

```diff
@@ -4,26 +4,27 @@ import PropTypes from 'prop-types';
 import ListItemStatus from '../ListItemStatus';
 import Dashboard from '../../../Dashboard';
 
-const ListItem = ({ name, statuses, initExpanded }) => (
+const ListItem = ({ accountID, statuses, initExpanded }) => (
   <ListView.Item
     leftContent={<ListView.Icon name="user" />}
-    heading={name}
+    heading={statuses.label}
```

Comment:
Can you rename the `statuses` prop name to `accountData` here too?
so `accountData.label` would make more sense.